EIPS/eip-5593.md

```diff
@@ -60,7 +60,7 @@ Wallet extensions SHOULD consider adding a "developer mode" toggle via a UX so t
 - Top level `https://b.com` with `<iframe src="https://a.com">` with `<iframe src="https://b.com">` -> blocked (third-party iframe without sufficient privileges)
 - Top level `https://a.com` with `<iframe src="https://sub.a.com">` -> blocked (third-party iframe without sufficient privileges)
 - Top level `https://a.com` with `<iframe src="https://a.com" sandbox>` -> blocked (sandbox attribuute without "allow-same-origin")
-- Top level `https://a.com` with `<iframe src="https://a.com" sandbox="allow-same-origin allow-scripts">` -> allowed (but note this case is discouraged in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox because it’d allow the iframe to remove its own sandbox attribute)
+- Top level `https://a.com` with `<iframe src="https://a.com" sandbox="allow-same-origin allow-scripts">` -> allowed (but note this case is discouraged because it'd allow the iframe to remove its own sandbox attribute)
```

Comment:
How is an implementer expected to better understand why this is a bad idea? Is it just because this EIP declares it so or is it because this was a tradeoff made in the sandbox attribute design?